app/assets/stylesheets/recipe_show.scss

```diff
@@ -0,0 +1,70 @@
+@import "colors";
+
+.recipe-info-block {
+  padding-bottom: 20px;
+}
+
+.recipe-info-attr {
+  background-color: $dark-orange;
+  border-radius: 5px;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);
+  padding: 0 5px;
+}
+
+.category-name {
+  background-color: $light-orange;
+  border-radius: 5px;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);
+  padding: 0 5px;
+}
+
+.ingredient-row {
+  padding-bottom: 20px;
+}
+
+.ingredient-block {
+  background-color: $light-green;
+  border-color: darken($light-yellow, 10%);
+  border-radius: 5px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 4px 4px 2px darken($light-yellow, 70%);
+  margin-left: 10px;
+  padding: 5px;
+}
+
+.ingredient-listing {
+  padding: 2px 0;
+}
+
+.step-block {
+  background-color: $light-green;
+  border-color: darken($light-yellow, 10%);
+  border-radius: 5px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 4px 4px 2px darken($light-yellow, 70%);
+  margin: 0 10px 20px 0;
+  padding: 5px;
+}
+
+.recipe-step-index {
+  font-family: 'Lobster', cursive;
+  font-size: 14pt;
+  margin-left: 5px;
+}
+
+.recipe-step {
+  background-color: $light-blue;
+  border-color: darken($light-green, 10%);
+  border-style: solid;
```

Comment:
Properties should be ordered background-color, border-color, border-radius, border-style, border-width, box-shadow, margin, padding